service/service.hpp

```diff
@@ -118,42 +130,47 @@ namespace raven
                     order_registry
                     {
                         {
-                            "CONFIG_CREATE",  [this](json::json &json_data) {
+                            "CONFIG_CREATE",     [this](json::json &json_data) {
                             this->create_config(json_data);
                         }},
                         {
-                            "CONFIG_LOAD",    [this](json::json &json_data) {
+                            "CONFIG_LOAD",       [this](json::json &json_data) {
                             this->load_config(json_data);
                         }},
                         {
-                            "CONFIG_UNLOAD",  [this](json::json &json_data) {
+                            "CONFIG_UNLOAD",     [this](json::json &json_data) {
                             this->unload_config(json_data);
                         }},
                         {
-                            "CONFIG_INCLUDE", [this](json::json &json_data) {
+                            "CONFIG_INCLUDE",    [this](json::json &json_data) {
                             this->include_config(json_data);
                         }},
                         {
-                            "SETTING_UPDATE", [this](json::json &json_data) {
+                            "SETTING_UPDATE",    [this](json::json &json_data) {
                             this->update_setting(json_data);
                         }},
                         {
-                            "SETTING_REMOVE", [this](json::json &json_data) {
+                            "SETTING_REMOVE",    [this](json::json &json_data) {
                             this->remove_setting(json_data);
                         }},
                         {
-                            "SETTING_GET",    [this](json::json &json_data) {
+                            "SETTING_GET",       [this](json::json &json_data) {
                             this->get_setting(json_data);
                         }},
                         {
-                            "ALIAS_SET",      [this](json::json &json_data) {
+                            "ALIAS_SET",         [this](json::json &json_data) {
                             this->set_alias(json_data);
                         },
                         },
                         {
-                            "ALIAS_UNSET",    [this](json::json &json_data) {
+                            "ALIAS_UNSET",       [this](json::json &json_data) {
                             this->unset_alias(json_data);
                         },
+                        },
+                        {
+                            "SUBSCRIBE_SETTING", [this](json::json &json_data) {
```

Comment:
as you can see the editor algned every lambda here because "SUBSCRIBE_SETTING" is more long than another